common/djangoapps/student/forms.py

```diff
@@ -3,6 +3,9 @@
 """
 from importlib import import_module
 import re
+import socket
```

Comment:
Please do not make any more changes to this old learner registration view. This view is only used by MITx site as far as we know and we don't think they will need this functionality. We are trying to get MITx to use the new login and registration pages so we can retire this view. Please consider the code here obsolete.